src/Screens/MainPage/components/SearchMainPage.tsx

```diff
@@ -10,13 +10,22 @@ import {FormControl } from "@mui/material";
 import { VerticalLine } from "../../../Comps/search/style";
 import DropDown from "../../../Comps/select/DropDown";
 import {Container} from './style'
-
+import RecentSearches from "../../../Comps/search/recentSearches/RecentSearches";
 
 export const SearchMainPage = () => {
   const [input, setInput] = useState<string>("");
+  const [items, setItems] = useState<string[]>([]);
+  const [focus, setFocus] = useState<boolean>(false);
```

Comment:
if you need to know if an input is focused or not you don't need to maintain a state, you can use - useRef